.gitignore

```diff
@@ -9,8 +9,8 @@ doc/doxygen/man
 doc/doxygen/*.log
 doc/doxygen/*.db
 doc/doxygen/*.tmp
-# Built binaries
-*bin
+# bin (e.g.:build directory)
+bin
```

Comment:
Unrelated?

Reply:
Ah there is an explicit commit, but why is round-robin ignored?

Reply:
no it is round_rob*bin*

Reply:
Ah duh got it, what about `*.bin`?

Reply:
Or better both?

Reply:
```suggestion
# bin (e.g.:build directory)
bin
*.bin
```

Reply:
will add that